Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Validator.rst

```diff
@@ -1,35 +1,37 @@
-.. include:: /Includes.rst.txt
+:navigation-title: Validator
 
-.. index:: Extbase; Validator
+..  include:: /Includes.rst.txt
+..  index:: Extbase; Validator
+..  _extbase_domain_validator:
 
-.. _extbase_domain_validator:
+=========================
+Custom Extbase Validators
+=========================
 
-=========
-Validator
-=========
-
-.. seealso::
-   *  :ref:`extbase_validation` for general validation in Extbase.
+..  seealso::
+    *  :ref:`extbase_validation` for general validation in Extbase.
 
 Custom validators are located in the directory :file:`Classes/Domain/Validator`
-and therefore in the namespace :php:`Vendor\MyExtension\Domain\Validator`.
+and therefore in the namespace :php:`MyVendor\MyExtension\Domain\Validator`.
+
+All validators must implement :php-short:`TYPO3\CMS\Extbase\Validation\Validator\ValidatorInterface`.
+The usually extend the :php-short:`\TYPO3\CMS\Extbase\Validation\Validator\AbstractValidator`.
 
-All validators extend the :php:`AbstractValidator`
-(:php:`\TYPO3\CMS\Extbase\Validation\Validator\AbstractValidator`).
+..  note::
+    In the package :php:`\TYPO3\CMS\Extbase\Validation\Validator\*` Extbase
```

Comment:
"package" is more composer package like typo3/cms-exbase. I would prefer using folder or directory:

```suggestion
    In the directory :php:`\TYPO3\CMS\Extbase\Validation\Validator\*` Extbase
```